_layouts/default.html

```diff
@@ -40,7 +40,7 @@
 					</button>
 				</li>
 				<li class="hidden-sm hidden-md hidden-lg">
-					<button id="menu-btn" title="Menu" class="collapsed" data-toggle="collapse" data-target="#mobile-menu" aria-expanded="false">
+					<button id="menu-btn" title="Menu">
```

Comment:
Can you port the `aria-expanded` behavior over as well?

Reply:
I thought about it, but what mobile browser/software supports these?

Reply:
Not sure, it's an ARIA attribute, so probably VoiceOver and/or JAWS and/or other assistive technology.